app/views/catalog/facet.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="facet_pagination top">
+  <%= render :partial=>'facet_pagination' %>
+</div>
+
+<div class="modal-header">
+  <button type="button" class="ajax-modal-close close" data-dismiss="modal" aria-hidden="true">×</button>
+
+  <h3 class="modal-title"><%= facet_field_label(@facet.key) %></h3>
+  <%= render partial: 'facet_index_navigation' if @facet.index_range && @display_facet.index? %>
+
+</div>
+<div class="modal-body">
+  <div class="facet_extended_list">
+  <!-- nocommit; @mksndz, please review? -->
+  <% facet_type = @facet[:facet_type] || :default %>
+  <% facet_type = facet_type.call(params) if facet_type.respond_to?(:lambda?) %>
+  <% puts "XXX params=#{params}" %>
+  <%= render_facet_limit(@display_facet, layout: false, facet_type: facet_type) %>
```

Comment:
this same logic to determine facet_type is done within the `render_facet_limit`. i don't see where this option is consumed by any of this new `subject_correlation` (the facet has its own partials for display) code, so why is this template override being introduced here?

Reply:
Yes it's weird. This was an afterthought -- and a bug that existed until fixed in 9fcde3f. But now I think I fixed it the wrong way! By the time you call `render_facet_limit`, you know you want the display and you don't care what the `facet_type` is. I can't see anywhere _but_ here that is calling `render_facet_limit`. So my best guess is that checking `facet_type` in `render_facet_limit` is vestigial, and should just have been removed, instead of jumping through hoops to pass in something irrelevant.
Also if by chance removing this _does_ break something, it's view code, so any issues will be readily apparent (i.e. not insidiously adding heavy unused facets to solr request). I'll push a commit momentarily. Thanks!